data-tck/src/main/java/io/micronaut/data/tck/repositories/BookRepository.java

```diff
@@ -81,8 +81,11 @@ public BookRepository(AuthorRepository authorRepository) {
     @Join("author")
     public abstract Book findByTitle(String title);
 
+    @Join(value = "author.books", type = Join.Type.LEFT_FETCH)
     public abstract Author findAuthorById(@Id Long id);
 
+    public abstract List<Book> findAuthorBooksById(@Id Long id);
```

Comment:
`@Id` annotation is not needed in this use-case.

Reply:
It was already in prev method, but I removed it.